.golangci.yml

```diff
@@ -57,7 +57,7 @@ linters-settings:
       - G306 # Poor file permissions used when writing to a new file.
 
   staticcheck:
-    go: "1.21"
+    go: "1.22.5"
```

Comment:
Since we're not updating the linter's base image and Golang version, we probably don't need to bump this? Doesn't seem to break anything, so non-blocking.